tester_app/client/app/bundles/HelloWorld/store/helloWorldStore.jsx

```diff
@@ -0,0 +1,24 @@
+import { compose, createStore, applyMiddleware, combineReducers } from 'redux';
+import thunkMiddleware from 'redux-thunk';
+import loggerMiddleware from 'lib/middlewares/loggerMiddleware';
```

Comment:
// This provides an example of logging redux actions to the console.
// You'd want to disable this for production.

@alexfedoseev -- probably should make this change.